contrib/ruby/test/client_test.rb

```diff
@@ -833,6 +839,10 @@ def test_configured_max_packet_above_server
     end
 
     refute_match(/TRILOGY_MAX_PACKET_EXCEEDED/, exception.message)
+
+    exception = assert_raises_connection_error do
```

Comment:
Tiniest nit -- let's avoid reassigning here?
```suggestion
    assert_raises_connection_error do
```